.github/workflows/ci.yaml

```diff
@@ -12,28 +12,38 @@ jobs:
         id: info
         shell: bash
         run: |
+          echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
           echo "##[set-output name=node_version;]$(cat .nvmrc)"
+          echo "##[set-output name=yarn_cache;]$(yarn cache dir)"
+          git config user.name "GitHub Actions Build"
+          git config user.email "developers@mobilehead.com"
 
       - id: use_node
         uses: actions/setup-node@v1
         with:
           node-version: ${{ steps.info.outputs.node_version }}
 
-      - name: ci_cache
-        id: ci_cache
+      - name: yarn_cache
+        id: yarn_cache
+        uses: actions/cache@v1
+        with:
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+          path: ${{ steps.info.outputs.yarn_cache }}
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+
+      - name: node_modules_cache
         uses: actions/cache@v1
         with:
-          key: ci_cache-${{ hashFiles('**/yarn.lock') }}
           path: node_modules
+          key: ${{ runner.OS }}-node_modules_cache-${{ hashFiles('**/yarn.lock') }}
```

Comment:
why do you have the os name in the cache key?

Reply:
idk, because it was in the examples (i guess it only matters when you run on different OSes via matrix)